addons/crm/models/crm_team_member.py

```diff
@@ -207,6 +207,6 @@ def _get_assignment_quota(self, work_days=1):
         :param float work_days: see ``CrmTeam.action_assign_leads()``;
         """
         assign_ratio = work_days / 30.0
-        to_assign = self.assignment_max * assign_ratio
+        to_assign = max(1, self.assignment_max * assign_ratio)
```

Comment:
That's wrong; it means you might assign 1 lead to someone who should receive 0 lead.

Reply:
:thinking: Yes, now that you mention it, it's not fair if you can get assigned leads even though you've already exceeded the max...

fp's proposition is indeed more in line with the rest of the assignment method (which assigns weights 0 on dropping a team member from the computation).
Users will get feedback saying nobody was assigned anyway so it's fine not to assign anyone.